src/ocl/convolutionocl.cpp

```diff
@@ -606,8 +606,7 @@ void ConvolutionDescriptor::ConvolutionForward(Handle& handle,
     ValidateConvTensors(tensors);
     ValidateAlphaBeta(alpha, beta);
 
-    if(algo != miopenConvolutionFwdAlgoGEMM &&
-       (xDesc.GetType() == miopenInt8 || xDesc.GetType() == miopenInt8x4))
+    if(algo != miopenConvolutionFwdAlgoGEMM && xDesc.GetType() == miopenInt8x4)
```

Comment:
Hi @jerryyin, as discussed in PR #1532 , we decide to decouple the data type and vector length, which means `miopenInt8x4`  may be discarded in the near future. Thus change the return value to `xDesc.GetType() == miopenInt8` here will be appreciate.

Reply:
That sounds like an orthogonal change with the diff here. This diff touched only on the applicable range of `miopenInt8` and preserved the original logic for `miopenInt8x4`. 

> which means miopenInt8x4 may be discarded in the near future

There should be another future PR that get rid of `miopenInt8x4` rather than me doing patch work in this MLIR pull request. By the time #1532 landed, we should not touch any code preemptively in order not to break any existing functionalities.